wao/error_watcher.py

```diff
@@ -39,14 +40,10 @@ def smooth_romeo_restart(error_line):
     error_line = "[REPORT TO TRELLO]" + error_line
     error_line += (" [SENDING SS]" if is_romeo_alive else " [POOL EMPTY. NO ROMEO FOUND]")
     post_request(error_line, is_from_error_handler=True)
+    romeo.send_error_report(error_line)
 
     if is_romeo_alive:
         romeo.perform_sell_signal(RomeoExitPriceType.SS)
-        # romeo.send_error_report(error_line) #todo
-    # else:
-    #     post_request(error_line) #todo uncomment
```

Comment:
there's remaining work here

Reply:
> there's remaining work here

i deliberately commented these out here because there was an issue. Regarding the missing functionality of new error report content being sent to telegram logs as well as trello, this structure of code wasn't able to do it. It simply gets skipped. However, when i took out romeo.send_error_report(error_line) from the if condition and tested the issue was fixed. We got a trello ticket with the new error content and also a telegram log. You can see the result of moving the 'romeo.send_error_report(error_line)' by following these 2 links
1. https://t.me/c/1589644762/159
2. https://trello.com/c/ZGGCi2KA/325-report-to-trelloerror-this-is-a-trello-error-ticket-created-from-code-sending-ss

Reply:
that's normal, you were sending buy signal from controller_test, so the pool was empty

revert... in production, it will work with the commented changes

Reply:
understood. Resolved.